utils/combinedLocales.js

```diff
@@ -3,7 +3,7 @@ import fs from 'fs';
 
 const pathToCombinedTranslationKeys = path.join(
   __dirname,
-  'combinedTranslationKeys.txt'
+  'combinedLocales.txt'
```

Comment:
Sorry to nitpick but as @alexcjohnson suggested perhaps we should continue calling these `translation-references` or `translation-strings` or something. Locales are the full objects with all the fancy dateFormats and such right?

Reply:
plotly.js calls it's file translation-keys, I could call it translation-keys for the editor too, and combined-translation-keys for the combined file, works?

Reply:
plotly.js has a rule of all-lowercase filenames, which seems to have fallen out of fashion in more recent js projects, but I won't vote against consistency across repos, I like your proposed names @VeraZab.

Reply:
sg!